plugins/UsersManager/lang/en.json

```diff
@@ -18,9 +18,10 @@
         "AuthTokenPurpose": "What are you using this token for?",
         "NoTokenCreatedYetCreateNow": "No token created yet, %1$screate a new token now%2$s.",
         "TokenSuccessfullyGenerated": "Token successfully generated",
-        "ConfirmTokenCopied": "I confirm I copied the token.",
+        "ConfirmTokenCopied": "I confirm I copied the token and understand I should not share this token with anyone else.",
```

Comment:
Not sure if that text might not get a bit too long when translated to other languages. Current layout will cause the text to be cut of instead of having a multiline button...

Reply:
Made it multiline 👍